src/lwc/geModalPrompt/__tests__/geModalPrompt.test.js

```diff
@@ -0,0 +1,66 @@
+import { createElement } from 'lwc';
+import GeModalPrompt from 'c/geModalPrompt';
+
+describe('c-ge-modal-prompt', () => {
```

Comment:
Nioce! 👍

Reply:
It is a humble thing, but I am very proud!